contrib/forms/xmlformgen/xslt/report.php.xslt

```diff
@@ -107,7 +107,7 @@ $lists = array(]]></xsl:text>
               $value = $dateparts[0];
             }
 
-	    echo "<td><span class='bold'>";
+	    echo "<td style='width:24%'><span class='bold'>";
```

Comment:
To confirm:
1. Does this also look ok in the html report output (since that has 6 columns) ?
2. Does this look ok in the encounter view output (since that has 2 columns) ?

If so, then will bring this into the codebase.
thanks, -brady